camel/model_backend.py

```diff
@@ -69,8 +71,10 @@ def run(self, *args, **kwargs) -> Dict[str, Any]:
         num_max_token = num_max_token_map[self.model_type.value]
         num_max_completion_tokens = num_max_token - num_prompt_tokens
         self.model_config_dict['max_tokens'] = num_max_completion_tokens
+        time.sleep(0.1)
```

Comment:
Why are you adding a sleep here?

Reply:
Oops, that can go. If there is a problem with the request, you instantly burn through your quota on Azure. A small delay let's you exit the program with hammering the servers.   The correct way is to step back the request rate exponentially.